images/Makefile

```diff
@@ -2,18 +2,20 @@
 WHAT?=
 DOCKER_USER?=weaveworks
 VERSION?=$(shell git describe HEAD --tags)
+RELEASE?=latest
+TAG:=${RELEASE}$(if $(strip $(VERSION)),-${VERSION})
 
 all: build
 build:
 ifeq ($(WHAT),)
 	$(error WHAT is a required argument)
 endif
 	@ls ${WHAT} >/dev/null
-	docker build -t ${DOCKER_USER}/ignite-${WHAT}:${VERSION} ${WHAT}
+	docker build --build-arg RELEASE -t ${DOCKER_USER}/ignite-${WHAT}:${TAG} ${WHAT}
 
 push:
-	docker push ${DOCKER_USER}/ignite-${WHAT}:${VERSION}
-	docker tag ${DOCKER_USER}/ignite-${WHAT}:${VERSION} ${DOCKER_USER}/ignite-${WHAT}:latest
+	docker push ${DOCKER_USER}/ignite-${WHAT}:${TAG}
+	docker tag ${DOCKER_USER}/ignite-${WHAT}:${TAG} ${DOCKER_USER}/ignite-${WHAT}:latest
```

Comment:
also tag `TAG` => `RELEASE` and push, so that e.g. `:cosmic` always points to the latest version

Reply:
only tag it latest if the (new) env `IS_LATEST==true`

Reply:
in other words, create a new `IS_LATEST=false` variable.
if it's set, also tag and push it as latest

set this `IS_LATEST` variable below, for the image combinations that it makes sense for (e.g. ubuntu:18.04, etc.)

each image should be tagged both `:18.04-v0.4.x`, and `:18.04`

Reply:
Done.